code/modules/mining/artifact/powers.dm

```diff
@@ -925,10 +925,11 @@
 			user.visible_message("<span class='danger'>[user] fires [src]!</span>", "<span class='danger'>You fire [src]!</span>", "You hear a burst of energy!")*/
 		//if(A_MAGICPROJ) -- TODO
 		if(A_TRAVEL)
-			var/list/pos_levels = list(1,1,1,3,4,5,6) // Travels to the station the majority of the time.
-			for(var/num in pos_levels) // Don't travel to the same z-level
-				if(target.z == num)
-					pos_levels -= num
+			var/list/pos_levels = list()
+			if(user.z == 1)
+				pos_levels = list(3,4,5,6,7)
```

Comment:
please use the Zlevel defines for these. See: https://github.com/tgstation/-tg-station/blob/cf6eca2d472b6e8c2ca80b7ba9f18678a8068173/code/__DEFINES/misc.dm#L115